static/js/ajax-notifications.js

```diff
@@ -39,7 +42,7 @@ function wsCheckNotifications() {
 
         // Add handling for different kinds of events. For ex: {"event": "notification", "data": {"title": ..}}
         if (msg.event === 'notification') {
-            displayPNotify(msg.data.type, msg.data.title, msg.data.body);
+            displayPNotify(msg.data.type, msg.data.title, msg.data.body, msg.data.hash);
         } else {
             displayPNotify('info', '', msg);
```

Comment:
I just figured it out.
You changed in the wrong location (this is not the websocket implementation)

You need to change it in [`medusa/ui.py`](https://github.com/pymedusa/Medusa/blob/feature/issue_2897/medusa/ui.py) in both [`Notifications.message()`](https://github.com/pymedusa/Medusa/blob/feature/issue_2897/medusa/ui.py#L47-L50) and [`Notifications.error()`](https://github.com/pymedusa/Medusa/blob/feature/issue_2897/medusa/ui.py#L59-L63)
(but you should leave this here, too)